### PR TITLE
Updated to Vapor 2

### PR DESCRIPTION
**Pull request by BrettRToomey**

### CODE REVIEW

File: Tests/BugsnagTests/PayloadTransformerTests.swift [hunk 1]
```diff
@@ -60,11 +60,11 @@ class PayloadTransformerTests: XCTestCase {
         let expectedJsonParams = Node(["json": "value"])
 
         XCTAssertEqual(request?["method"]?.string, "GET")
-        XCTAssertEqual(request?["headers"]?.node, expectedHeaders)
-        XCTAssertEqual(request?["urlParameters"]?.node, expectedUrlParams)
-        XCTAssertEqual(request?["queryParameters"]?.node, expectedQueryParams)
-        XCTAssertEqual(request?["formParameters"]?.node, expectedFormParams)
-        XCTAssertEqual(request?["jsonParameters"]?.node, expectedJsonParams)
+        XCTAssertEqual(request?["headers"]?.makeNode(in: nil), expectedHeaders)
```
Comment (tanner0101) [inline Tests/BugsnagTests/PayloadTransformerTests.swift:63]: you might be able to use `.wrapped` here

Comment (BrettRToomey) [reply Tests/BugsnagTests/PayloadTransformerTests.swift:63]: Ah right. Let me try!

**Comment by BrettRToomey** [comment]
Updating Travis script for Swift 3.1

File: Tests/BugsnagTests/PayloadTransformerTests.swift (hunk 1 above)
Comment (BrettRToomey) [reply Tests/BugsnagTests/PayloadTransformerTests.swift:63]: Yup, it works.

**Comment by steffendsommer** [comment]
LGTM - adding a new branch (`vapor-1`) and re-routing the other PRs into that

**Pull request merged**